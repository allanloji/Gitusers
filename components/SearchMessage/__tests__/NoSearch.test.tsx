import React from 'react';
import { render } from '@testing-library/react';

import NoSearch from '..';

describe('NoSearch', () => {
  test('Should render', () => {
    const { container } = render(<NoSearch title='Test message' image='ðŸ‘€' />);
    expect(container).toMatchSnapshot();
  });
});
