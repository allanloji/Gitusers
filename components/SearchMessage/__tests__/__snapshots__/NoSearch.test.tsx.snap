// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoSearch Should render 1`] = `
<div>
  <div
    class="css-1dszs7t"
  >
    <span
      class="css-gjmu0c"
    >
      ðŸ‘€
    </span>
    <h1
      class="css-17s2ops"
    >
      Test message
    </h1>
  </div>
</div>
`;
